### PR TITLE
Issue 374 update extreme events

### PR DESCRIPTION
**Pull request by mslarae13**
#374 
added extreme_event to slot usage for biosample.
Edited MIxS format requirments / syntax and provided an example

### CODE REVIEW

**Comment by mslarae13** [comment]
Pending approval, submit a ticket to GSC

File: src/schema/nmdc.yaml [hunk 1]
```diff
@@ -476,6 +474,24 @@ classes:
         required: false
       part_of:
         required: true
+      extreme_event:
```
Comment (mslarae13) [inline src/schema/nmdc.yaml:477]: in mixs.yaml there's also 
"
    from_schema: http://w3id.org/mixs/terms
    source: http://w3id.org/mixs/terms
    see_also:
      - https://github.com/microbiomedata/nmdc-schema/blob/issue-291-mixs-submod/util/rebuild_mixs_yaml.py
   "
@turbomam  should I have left that in there?

**Comment by mslarae13** [comment]
Remove repeated information. Anything that is inherited from the parent slot & not changed in usage should not be re-listed. 

Don't edit 'aliases'

Update "range: text" to range: string

File: src/schema/nmdc.yaml (hunk 1 above)
Comment (turbomam) [reply src/schema/nmdc.yaml:527]: The new content and your following of SOPs are really good. Here's to many more.

**Review by turbomam — APPROVED** [review]

**Pull request merged**